tests/testthat/setup-fakedata.R

```diff
@@ -198,12 +199,13 @@ tt_to_test_newline_chars <- function() {
       split_label = "m\nannaggia\nsda\n",
       label_pos = "visible"
     ) %>%
+    split_cols_by("ARM") %>%
```

Comment:
why did we need this? It's just an extra test right?

Reply:
yes it is an additional test